controllers/external/tracker.go

```diff
@@ -56,15 +57,15 @@ func (o *ObjectTracker) Watch(log logr.Logger, obj runtime.Object, handler handl
 	u := &unstructured.Unstructured{}
 	u.SetGroupVersionKind(gvk)
 
-	log.Info("Adding watcher on external object", "groupVersionKind", gvk.String())
+	log.Info(fmt.Sprintf("Adding watch on external object %q", gvk.String()))
```

Comment:
I think `log.Infof` gets us the `%q` that we want here without having to import `fmt`

Reply:
logr.Logger doesn't have Infof